### PR TITLE
[Approaches]: Leap

### PR DESCRIPTION
**Pull request by glaxxie**

### CODE REVIEW

File: exercises/practice/leap/.approaches/boolean-chain/content.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+First we test if the year is divisible by 4 using modulus, if it return `false` here then the chain is short circuit and return `false`.
+If the year is indeed divisible by 4, then we test whether it can't divisible by 100 or can divisible by 400.
+If one of those condition return `true` then our chain will return `true`, but if both return `false` then our chain here will return `false`.
+
+You can visualize the chain like this:
+
+divisible by 4          | not divisible by 100 | divisible by 400   |
+false (short circuit)   |                      |                    | => false 
+true                    |      only one of these need to be true    | => true
+true                    |   false              |   false            | => false
```
Comment (ErikSchierboom) [inline exercises/practice/leap/.approaches/boolean-chain/content.md]: It looks like this table doesn't render correctly.

**Review by ErikSchierboom — COMMENTED** [review]
One tiny nit

File: exercises/practice/leap/.approaches/boolean-chain/content.md (hunk 1 above)
Comment (glaxxie) [reply exercises/practice/leap/.approaches/boolean-chain/content.md]: I actually re-did it before i push, not sure why it didnt save properly. Let me try again

Comment (glaxxie) [reply exercises/practice/leap/.approaches/boolean-chain/content.md]: Should be good now!

**Review by ErikSchierboom — APPROVED** [review]
Lovely work!
